src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -416,17 +435,7 @@ public async Task UnconfigureAsync(CommandSettings command)
                     _term.WriteLine("Does not exist. Skipping " + currentAction);
                 }
 
-                //delete settings config file
-                currentAction = "Removing .runner";
-                if (isConfigured)
-                {
-                    _store.DeleteSettings();
-                    _term.WriteSuccessMessage("Removed .runner");
-                }
-                else
-                {
-                    _term.WriteLine("Does not exist. Skipping " + currentAction);
-                }
+                DeleteLocalRunnerConfig();
```

Comment:
we better include `//delete credential config files` part to this as well since those are also staled info for the deleted runner.

Reply:
Done in 8e8458b